src/css/style.css

```diff
@@ -104,3 +109,80 @@ img {
 .card:hover {
     transform: scale(1.1)
 }
+
+
+
+/* code for service_cards page */
+
+.services_heading {
+display: flex;
+justify-content: center;
+font-weight: bolder;
+font-size: xx-large;
+margin-top: 40px;
+margin-bottom: 40px;
+}
+
+.servicesBox {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;}
```

Comment:
try without align items center - I think it looks better. 
and put the closing brace on its own line.

Reply:
hmmm....difference of opinion on the sans-align items....we shall have to roshambo